### DISCUSSION TITLE
ASLR and DEP enabled version of BouncyCastle.Crypto.dll

### DISCUSSION THREAD

**Discussion by NSasidharan** [bcgit/bc-csharp#384] [Q&A]
Hi,

We would like to know if you have any version of BouncyCastle.Crypto.dll that has **DEP** (Data Execution Prevention) and **ASLR** (Address Space Layout Randomization) enabled.

**Comment by bmmaxfield** [comment]
Hi, 

Any updates regarding or response to the question above?
